Doc/library/socket.rst

```diff
@@ -200,6 +200,16 @@ created.  Socket addresses are represented as follows:
 
   .. versionadded:: 3.8
 
+- :const:`IPPROTO_UDPLITE` is a variant on UDP which allows you to specify
+  what portion of a packet to cover with the checksum. It adds the methods
+  :meth:`socket.set_send_checksum_coverage` to change what portion of
+  outgoing packets are covered and :meth:`socket.set_recv_checksum_coverage`
+  to filter out packets which cover too little of their data.
+
+  ..availability:: Linux >= 2.6.20
```

Comment:
It should be `.. availability::`. Therefore the tests don't pass :)

Reply:
Should be fixed in most recent commits